mkdd_editor.py

```diff
@@ -2250,6 +2250,21 @@ def button_side_button_action(self, option, obj=None):
             self.object_to_be_added = [libbol.KartStartPoint.new(), -1, -1]
             self.pik_control.button_add_object.setChecked(True)
             self.level_view.set_mouse_mode(mkdd_widgets.MOUSE_MODE_ADDWP)
+        elif option == "route_object":
+            new_route = libbol.Route.new()
+            forward, up, left = obj.rotation.get_vectors()
+
+            new_point_1 = libbol.RoutePoint.new()
+            new_point_1.position = obj.position + left * 500
+            new_route.points.append(new_point_1)
+
+            new_point_2 = libbol.RoutePoint.new()
+            new_point_2.position = obj.position + left * -750
+            new_route.points.append(new_point_2)
+            self.action_ground_objects( [point.position for point in (new_point_1, new_point_2)] )
```

Comment:
Nit: Some stray spaces here.

```diff
- self.action_ground_objects( [point.position for point in (new_point_1, new_point_2)] )
+ self.action_ground_objects([point.position for point in (new_point_1, new_point_2)])
```

...although I'd probably suggest this, which happens to be shorter and should be more readable):

```python
self.action_ground_objects((new_point_1.position, new_point_2.position))
```